### PR TITLE
Add some more ignition examples

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

File: ignition/examples.md [hunk 1]
```diff
@@ -65,6 +67,57 @@ data and refuse to overwrite it.
...
+					]
+				}
+			}
+		]
+	}
+}
+```
+
+The create options are forwarded as-is to the underlying mkfs.$format
+utility, their respective man pages document what's appropriate here.
```
Comment (robszumski) [inline ignition/examples.md]: document**s the available settings.**

..or config
..or flags

File: ignition/examples.md [hunk 2]
```diff
@@ -65,6 +67,57 @@ data and refuse to overwrite it.
...
+
+## Create Files on the Root Filesystem ##
+
+Whether formatting a new filesystem or reusing the existing one, files may be
+created in the filesystem on the named device.  When not also formatting, care
+must be taken to specify the appropriate format.
```
Comment (robszumski) [inline ignition/examples.md]: Is this a fair rewrite of the last sentence?

> If you're using a formatted disk, be sure to supply that format when creating your files.

File: ignition/examples.md [hunk 3]
```diff
@@ -40,10 +40,12 @@ WantedBy=multi-user.target
 
 In many scenarios, it may be preferable to use btrfs for the root filesystem.
```
Comment (joshix) [inline ignition/examples.md]: Although CoreOS uses ext4 by default, the btrfs filesystem may be an appropriate choice for the system root partition in some scenarios.

File: ignition/examples.md [hunk 4]
```diff
@@ -40,10 +40,12 @@ WantedBy=multi-user.target
...
 This config will find the device with the filesystem label "ROOT" (the root
```
Comment (joshix) [inline ignition/examples.md]: "This example Ignition configuration will locate the device with the "ROOT" filesystem label..."

File: ignition/examples.md [hunk 5]
```diff
@@ -65,6 +67,57 @@ data and refuse to overwrite it.
...
+	}
+}
+```
+
+The create options are forwarded as-is to the underlying mkfs.$format
+utility, their respective man pages document what's appropriate here.
+
+## Create Files on the Root Filesystem ##
+
+Whether formatting a new filesystem or reusing the existing one, files may be
```
Comment (joshix) [inline ignition/examples.md]: s/the/an/

File: ignition/examples.md (hunk 2 above)
Comment (joshix) [inline ignition/examples.md]: When using an existing, already-formatted filesystem, be sure to supply its filesystem format (e.g., `ext4`, `btrfs`) in the configuration used to create files.

**Comment by vcaputo** [comment]
@joshix incorporated your changes, ptal

**Pull request merged**